scripts/utils/build.js

```diff
@@ -129,14 +129,17 @@ function buildCommonJS(pkgName, pkgPath) {
  * @returns {undefined}
  */
 function buildES(pkg) {
+  if (`${pkg}` === 'widget-demo' || `${pkg}` === 'widget-recents-demo'|| `${pkg}` === 'widget-space-demo') {
+    return;
```

Comment:
All demo files are failing the buildES method because they have SCSS styles. Also since we are dont publishing these on npmjs I think we dont need the this. Tried to run the the demos and they are working fine